### ISSUE TITLE
Is possible to add free Vazir font for Persian/Arabic content please?

### ISSUE THREAD

**Issue by hamedsbt** [FredJul/Flym#597]
https://github.com/rastikerdar/vazir-font

**Comment by mitsnosrap** [comment]
@hamedsbt Can you provide some sample feeds that would make use of this font? Thanks.

**Comment by hamedsbt** [comment]
Thank you for reply,

https://www.zoomit.ir/2020/6/1/359150/end-handshakes-humans-and-robots/

**Comment by mitsnosrap** [comment]
@hamedsbt Thanks for supplying that link. I have a few more questions:

1. It looks like the app is able to render the content of that feed (may vary from device to device). Does that feed render on your device?
2. Is this a request due to a preference to use a different font in the app?
3. Or is this a request because you are unable view the content / it doesn't get rendered properly?

In looking into alternative font support, it looks like it would be a considerable effort. If I were to implement this feature, I would probably look to support alternative fonts via the [downloadable font feature](https://developer.android.com/guide/topics/ui/look-and-feel/downloadable-fonts). That would require adding support for alternative downloadable fonts and making sure all the TextViews throughout the app have the alternative font applied.

**Comment by hamedsbt** [comment]
@mitsnosrap
Default android's font for Persian text is not comfortable for long time reading so I'd like the app support to change font family as default font. 
"Is this a request due to a preference to use a different font in the app?" YES!

Some Persian content RSS links:
http://techshot.ir/Feed/RSS
http://techtalks.ir/feed/
http://barsam.ir/feed/
http://www.panevesht.com/rss.xml
http://digi-land.ir/feed/
http://www.digiboy.ir/?feed=rss2
http://www.imarketor.com/feed/
http://toranji.ir/rss/
http://www.gooyait.com/feed
https://tech.tavaana.org/fa/rss.xml
http://feeds.feedburner.com/seocommunitypersian
http://feeds.1pezeshk.com/pezeshk
http://www.zoomtech.ir/feed/

**Comment by mitsnosrap** [comment]
@hamedsbt Thanks for the reply! I've seen other issues reported where people would like to switch the font to something else too, so this may be worth the effort.

**Comment by FredJul** [comment]
On my side I would definitively accept such a feature (at least if the code is nice enough)

**Comment by mitsnosrap** [comment]
@FredJul Yeah, before I invested a ton of time into it, I'd probably research it, document my findings and create a PR for you - outlining the approach I'd take. However, I'm not certain I'll tackle this feature.

As you noticed, I recently went through all the open issues. I created a spreadsheet where I tried to categorize them so I could see what are the most requested issues or most reported bugs. Custom fonts as a feature request is in the top 10, but I'm not sure that's what I'll work on.  I need to go back and look through my spreadsheet/tickets again.
